src/overview.md

```diff
@@ -183,27 +183,68 @@ To find additional APIs, see the
 [commonly used packages page](/guides/libraries/useful-libraries).
 
 
-## Dart: The platform
+## Dart: The platform {#platform}
 
 Dart's compiler technology lets you run code in different ways:
 
-* **Native compilation**: For apps targeting mobile and desktop devices,
+* **Native platform**: For apps targeting mobile and desktop devices,
   Dart includes both a Dart VM with just-in-time (JIT) compilation and
   an ahead-of-time (AOT) compiler for producing machine code.
 
-* **Web compilation**: For apps targeting the web,
+* **Web platform**: For apps targeting the web,
   Dart includes both a development time compiler (dartdevc) and
   a production time compiler (dart2js).
   Both compilers translate Dart into JavaScript.
 
 <img src="{% asset Dart-platforms.svg @path %}" width="800px" alt="An
 illustration of the targets supported by Dart">
 
-In addition, [Flutter]({{site.flutter}}) provides tooling and UI libraries to
-build UIs that run on iOS, Android, macOS, Windows, and Linux.
+The [Flutter framework]({{site.flutter}}) is a popular multi-platform UI toolkit
+that's powered by the Dart platform, and which provides tooling and UI libraries
+to build UI experiences that run on iOS, Android, macOS, Windows, Linux, and the
+Web.
 
+#### Dart Native (machine code JIT and AOT) {#native-platform}
 
-## Learning Dart
+During development, aa fast developer cycle is critical for iteration. The Dart
+VM offers a just-in-time compiler (JIT) that supports both pure interpretation
+(as required on iOS devices, for example) and runtime optimization.
```

Comment:
This sentence left me a little confused -- because the clause "runtime optimization" sounded like you were talking about the finished version of the app. I wonder if you should more heavily emphasize the developer-cycle benefits of a VM (specifically, being able to send deltas without recompiling the app, app profiling and observatory), instead of getting into the weeds of the mechanics of compilation during development.